_aggregations/metric/median-absolute-deviation.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: default
+title: Median Absolute Deviation
+parent: Metric aggregations
+grand_parent: Aggregations
+nav_order: 65
+redirect_from:
+  - /query-dsl/aggregations/metric/median-absolute-deviation/
+---
+
+# Median Absolute Deviation Aggregations
+
+The `median_absolute_deviation` metric is a single-value metric aggregation that returns median absolute deviation field. Median absolute deviation is a statistical measure of data variability. It is used to measure dispersion from the median, may be less impacted by outliers in a dataset. 
+
+Median absolute deviation is calculated with:<br>
+median_absolute_deviation = median(|X<sub>i</sub> - Median(X<sub>i</sub>)|)
+
+The following example calculates the median absolute deviation of the `DistanceMiles` field of the opensearch_dashboards_sample_data_flights index:
+
+
+```json
+GET opensearch_dashboards_sample_data_flights/_search
+{
+  "size": 0,
+  "aggs": {
+    "median_absolute_deviation_DistanceMiles": {
+      "median_absolute_deviation": {
+        "field": "DistanceMiles"
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+#### Example response
+
+```json
+{
+  "took": 35,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 10000,
+      "relation": "gte"
+    },
+    "max_score": null,
+    "hits": []
+  },
+  "aggregations": {
+    "median_absolute_deviation_distanceMiles": {
+      "value": 1829.8993624441966
+    }
+  }
+}
+```
+
+### Missing
+If there are documents in your index that are missing fields and you wish to set a default value for those documents, use the `missing` parameter to specify the default value for any missing fields from documents.
```

Comment:
To use the active voice as specified in https://github.com/opensearch-project/documentation-website/blob/main/STYLE_GUIDE.md#voice, we can modify to have:
"You can set a default value for missing fields from documents by specifying the `missing` parameter.

Does this also replace `NULL` values?

Reply:
For context, it also replaces NULL in bucket aggregations https://opensearch.org/docs/latest/aggregations/bucket/missing/

Reply:
Added details about null and added more active voice